
One type fix and the gettimeofday misdetection - the later could need some
in depth review.

  - Rene Rebe <reneÂ@exactcode.de>

diff -ur rsync-2.6.3pre2/lib/compat.c rsync-2.6.3pre2-diet/lib/compat.c
--- rsync-2.6.3pre2/lib/compat.c	2004-01-20 01:59:26.000000000 +0100
+++ rsync-2.6.3pre2-diet/lib/compat.c	2005-02-03 00:14:02.275505872 +0100
@@ -196,9 +196,5 @@
 /* some systems don't take the 2nd argument */
 int sys_gettimeofday(struct timeval *tv)
 {
-#if HAVE_GETTIMEOFDAY_TZ
 	return gettimeofday(tv, NULL);
-#else
-	return gettimeofday(tv);
-#endif
 }
diff -ur rsync-2.6.3pre2/rsync.h rsync-2.6.3pre2-diet/rsync.h
--- rsync-2.6.3pre2/rsync.h	2004-08-03 17:41:17.000000000 +0200
+++ rsync-2.6.3pre2-diet/rsync.h	2005-02-03 00:13:07.860778168 +0100
@@ -326,7 +326,7 @@
 #define uint64 unsigned long long
 #else
 /* As long as it gets... */
-#define uint64 unsigned off_t
+#define uint64 uint64_t
 #endif
 
 /* Starting from protocol version 26, we always use 64-bit
